crates/driver/src/print_build.rs

```diff
@@ -248,127 +248,271 @@ where
     };
 
     let target_folder = opt.working_directory.join(&element);
-    let target_name = target_folder
-        .file_name()
-        .unwrap()
-        .to_str()
-        .unwrap()
-        .to_string();
+    let base_name = to_file_name(&target_folder);
+    let mut t = TargetEntries {
```

Comment:
what about adding `derive(Default)` to `TargetEntries` so this is `Default::default()`

Reply:
Addressed in 662f6714d7713bb5d5b3d29da9cc3286b97ff33c